src/utils/config.ts

```diff
@@ -51,6 +51,11 @@ const configParsers = {
 
 		return parsed;
 	},
+	proxy(key: string) {
+		parseAssert('proxy', key.length === 0 || /^https?:\/\//.test(key), 'Must be a valid URL');
```

Comment:
I updated the logic here so if there's nothing provided, it returns the default value. So for this, you can probably just return if the argument is empty.